clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -2608,9 +2608,10 @@ RValue CIRGenFunction::emitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
   // can move this up to the beginning of the function.
   //   checkTargetFeatures(E, FD);
 
-  if ([[maybe_unused]] unsigned VectorWidth =
-          getContext().BuiltinInfo.getRequiredVectorWidth(BuiltinID))
-    llvm_unreachable("NYI");
+  if ([[maybe_unused]] unsigned vectorWidth =
+          getContext().BuiltinInfo.getRequiredVectorWidth(BuiltinID)) {
+    // Track the vector width - NYI.
```

Comment:
Seems like adding `LargestVectorWidth` is simple enough and shouldn't affect anything, so please go ahead and do it.